### ISSUE TITLE
Prepare questions for eLife data model discussion on Wednesday

### ISSUE THREAD

**Issue by g-sam** [elifesciences/elife-xpub#136]

**Comment by jsms90** [comment]
This morning we went through [the normal workflow diagram](https://drive.google.com/drive/u/1/folders/15NhjClwZUnl_x8TQPlj9dBRb4RIasOSx) & [the appeal workflow diagram](https://drive.google.com/drive/u/1/folders/15NhjClwZUnl_x8TQPlj9dBRb4RIasOSx), to cement an understanding of the world that we're trying to model. This threw up many questions, which I'm writing up now & will put in the google drive.

We've only just started discussing how we might actually model this in the data, but will have to continue this on Monday when we are next all together. We aim to produce a suggestion of the data model that we will use for initial submission, along with any further questions that we have.

Perhaps we can then use this data model suggestion + questions as a jumping off point for the 1.5 hour slot on Wednesday?
